src/instruments/src/EWD/elements/PseudoFWC.tsx

```diff
@@ -0,0 +1,1597 @@
+            memoInhibit: false,
+            failure: 0,
+            sysPage: -1,
+            side: 'RIGHT',
+        },
+        '0000040': // NW STRG DISC
+        {
+            flightPhaseInhib: [],
+            simVarIsActive: nwSteeringDisc === 1,
+            whichCodeToReturn: [engine1State > 0 || engine2State > 1 ? 1 : 0],
+            codesToReturn: ['000004001', '000004002'],
+            memoInhibit: false,
+            failure: 0,
+            sysPage: -1,
+            side: 'RIGHT',
+        },
+        '0000160': // PTU ON
+        {
+            flightPhaseInhib: [],
+            simVarIsActive: hydPTU === 1,
+            whichCodeToReturn: [0],
+            codesToReturn: ['000016001'],
+            memoInhibit: false,
+            failure: 0,
+            sysPage: -1,
+            side: 'RIGHT',
+        },
+        '0000210': // RAT OUT
+        {
+            flightPhaseInhib: [],
+            simVarIsActive: ratDeployed > 0,
+            whichCodeToReturn: [[1, 2].includes(flightPhase) ? 1 : 0],
+            codesToReturn: ['000021001', '000021002'],
+            memoInhibit: false,
+            failure: 0,
+            sysPage: -1,
+            side: 'RIGHT',
+        },
+        '0000070': // IGNITION
+        {
+            flightPhaseInhib: [],
+            simVarIsActive: engSelectorPosition === 2,
+            whichCodeToReturn: [0],
+            codesToReturn: ['000007001'],
+            memoInhibit: false,
+            failure: 0,
+            sysPage: -1,
+            side: 'RIGHT',
+        },
+        '0000540': // PRED W/S OFF
+        {
+            flightPhaseInhib: [1, 10],
+            simVarIsActive: !!(predWSOn === 0 && ![1, 10].includes(flightPhase)),
+            whichCodeToReturn: [[3, 4, 5, 7, 8, 9].includes(flightPhase) || toconfig === 1 ? 1 : 0],
+            codesToReturn: ['000054001', '000054002'],
+            memoInhibit: false,
+            failure: 0,
+            sysPage: -1,
+            side: 'RIGHT',
+        },
+        '0000545': // TERR OFF
+        {
+            flightPhaseInhib: [1, 10],
+            simVarIsActive: !!(gpwsOff === 1 && ![1, 10].includes(flightPhase)),
+            whichCodeToReturn: [[3, 4, 5, 7, 8, 9].includes(flightPhase) || toconfig === 1 ? 1 : 0],
+            codesToReturn: ['000054501', '000054502'],
+            memoInhibit: false,
+            failure: 0,
+            sysPage: -1,
+            side: 'RIGHT',
+        },
+        '0000320': // TCAS STBY
+        {
+            flightPhaseInhib: [],
+            simVarIsActive: !!(tcasMode === 0 && flightPhase !== 6),
+            whichCodeToReturn: [0],
+            codesToReturn: ['000032001'],
+            memoInhibit: false,
+            failure: 0,
+            sysPage: -1,
+            side: 'RIGHT',
+        },
+        '0000325': // TCAS STBY in flight
+        {
+            flightPhaseInhib: [],
+            simVarIsActive: !!(tcasMode === 0 && flightPhase === 6),
```

Comment:
See above - you could use tcasSensitivity instead if you want.

Reply:
Done